lib/oci_utils/migrate/image_types/qcow2.py

```diff
@@ -90,19 +91,17 @@ def __init__(self, filename):
         filename: str
             Full path of the qcow2 image file.
         """
-        _logger.debug('qcow2 header size: %d bytes' % self.head_size)
-        super(Qcow2Head, self).__init__(filename)
+        _logger.debug('qcow2 header size: %d bytes', self.head_size)
+        super().__init__(filename)
```

Comment:
call to super methods should be the first calls

Reply:
changed